workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/pages/Project/Attachments/components/TicketAttachmentsTable/_ticket_attachments_table.scss

```diff
@@ -35,6 +35,12 @@
 	}
 }
 
+.delete-ticket-attachment-modal {
+	.header-modal-title {
```

Comment:
If there is only one item in the nesting you can just consolidate into one line

Reply:
Done.